tests/schemas/invalid/test_invalid_placeholder_answer_type_from_transform.json

```diff
@@ -0,0 +1,103 @@
+{
+  "mime_type": "application/json/ons/eq",
+  "language": "en",
+  "schema_version": "0.0.1",
+  "data_version": "0.0.3",
+  "survey_id": "0",
+  "title": "Placeholder Test",
+  "theme": "default",
+  "description": "A questionnaire to test placeholders",
```

Comment:
This should explain why this is an invalid schema.

Reply:
Added explanation thanks!